src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -79,7 +79,7 @@ internal bool TryUnparse(StringBuilder sb)
         /// <summary>
         /// This class exposes <see cref="NgramExtractorTransform"/>/<see cref="NgramHashExtractingTransformer"/> arguments.
         /// </summary>
-        public sealed class Arguments : TransformInputBase
+        internal sealed class Arguments : TransformInputBase
```

Comment:
>Arguments [](start = 30, length = 9)

Can you rename to Options?

Reply:
If you notice in line #126 below, we have a public Options class, which the public API uses.  Hence I  kept it as is.  

DO u prefer calling it something else,  e.g. `OptionsInternal` ?  But it would only be cosmetic, since its 'internal'

---
In reply to: [253320489](https://github.com/dotnet/machinelearning/pull/2394#discussion_r253320489) [](ancestors = 253320489)